.go-version

```diff
@@ -1 +0,0 @@
-1.14.4
```

Comment:
I wasn't able to find any reason to keep this hidden file around

Reply:
Maybe it was to support [`goenv`](https://github.com/syndbg/goenv)? I used to use that, and I'm the only commiter on that file. 👍🏻 to deleting it.